rasa/core/events/__init__.py

```diff
@@ -632,17 +648,20 @@ def __eq__(self, other) -> bool:
         else:
             return self.name == other.name
 
-    def __str__(self) -> str:
+    def __str__(self) -> Text:
         return (
-            "ReminderScheduled("
-            "action: {}, trigger_date: {}, name: {}"
-            ")".format(self.action_name, self.trigger_date_time, self.name)
+            f"ReminderScheduled(intent: {self.intent}, trigger_date: {self.trigger_date_time}, "
+            f"entities: {self.entities}, name: {self.name})"
         )
 
+    def job_string(self) -> Text:
```

Comment:
optional: 
- avoid hungarian notation. 
- is there a name which better reflects the intent of this method?

Reply:
In some future PR we then probably should also rename all the `as_story_string` functions, right?